File: 백준/Bronze/2884. 알람 시계/알람 시계.c
#include <stdio.h>

int main() {
    int h = 0, m = 0, early = 45;
    scanf("%d %d", &h, &m);

    m -= 45;
    
    if (m < 0) {
        h -= 1;
        m += 60;

        if (h < 0) {
            h += 24;
        }
    }

    printf("%d %d", h, m);

    return 0;
}

File: 백준/Bronze/2884. 알람 시계/알람 시계.js
const fs = require('fs');

const readLine = fs.readFileSync(0, 'utf8').trim();

let [h, m] = readLine.split(' ').map(Number);
const early = 45;

m -= early;

if (m < 0) {
    h -= 1;
    m += 60;

    if (h < 0) {
        h += 24;
    }
}

console.log(h + " " + m);